package.json

```diff
@@ -25,71 +25,65 @@
   },
   "homepage": "https://github.com/blackbaud/skyux-sdk-builder#readme",
   "peerDependencies": {
-    "@angular/compiler-cli": "^7.2.1",
-    "@angular/compiler": "^7.2.1",
-    "@angular/common": "^7.2.1",
-    "@angular/core": "^7.2.1",
-    "@angular/forms": "^7.2.1",
-    "@angular/http": "^7.2.1",
-    "@angular/platform-browser": "^7.2.1",
-    "@angular/platform-browser-dynamic": "^7.2.1",
-    "@angular/router": "^7.2.1",
-    "@blackbaud/auth-client": "^2.21.0",
+    "@angular/compiler-cli": ">9.0.0-rc",
+    "@angular/compiler": ">9.0.0-rc",
+    "@angular/common": ">9.0.0-rc",
+    "@angular/core": ">9.0.0-rc",
+    "@angular/forms": ">9.0.0-rc",
+    "@angular/platform-browser": ">9.0.0-rc",
+    "@angular/platform-browser-dynamic": ">9.0.0-rc",
+    "@angular/router": ">9.0.0-rc",
+    "@blackbaud/auth-client": "^2.19.0",
     "@skyux-sdk/pact": "^3.1.0",
     "@skyux-sdk/testing": "^3.0.0",
     "@skyux/assets": "^3.1.0",
-    "@skyux/config": "^3.6.0",
+    "@skyux/config": "^3.7.0",
     "@skyux/core": "^3.10.0",
-    "@skyux/http": "^3.7.0",
-    "@skyux/i18n": "^3.5.0",
-    "@skyux/modals": "^3.0.0",
+    "@skyux/http": ">4.0.0-rc",
+    "@skyux/i18n": "^3.9.0",
     "@skyux/omnibar-interop": "^3.1.0",
-    "@skyux/router": "^3.1.0",
-    "@skyux/theme": "^3.4.0",
-    "@types/core-js": "^2.5.0",
-    "codelyzer": "^4.5.0",
-    "core-js": "^2.6.3",
-    "rxjs": "^6.3.3",
-    "rxjs-compat": "^6.3.3",
```

Comment:
Removed `rxjs-compat` as a peer dependency (consumers can still install it if they wish).